iacrcc/template.tex

```diff
@@ -68,6 +68,10 @@
            email   = {mccurley@digicrime.com},
           ]{Kevin S. \surname{McCurley}}
 
+% The following command controls the running header for authors
+% This is optional for <= 4 authors and mandatory for > 5 authors
```

Comment:
I think the comment should be:

% This is optional for <= 4 authors and mandatory for > 4 authors

Reply:
Yes, you are obviously correct.